climpred/metrics.py

```diff
@@ -11,6 +11,48 @@
     rmse,
 )
 
+ALL_HINDCAST_METRICS_DICT = {
```

Comment:
Constants should theoretically work at top level

Reply:
I agree. I am seeing on Travis though that it doesn't seem to think `_pearson_r` exists when referencing this dictionary. Maybe that's unrelated, or maybe that's the issue @aaronspring was seeing.

Reply:
Ah. The reason is that our dictionary references `_pearson_r` and so on, which is the name we use to wrap them. We don't define the `_pearson_r` and others until later on in `metrics`. So it tries to create this dictionary and those names don't exist. Hence why it has to be at the bottom. So we either need to keep it at the bottom or find a solution for the underscore naming convention.

Reply:
I made a constants.py